en/news/_posts/2019-04-23-move-to-git-from-svn.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news_post
+title: "Ruby Repository Moved to Git from Subversion"
+author: "hsbt"
+translator:
+date: 2019-04-23 00:00:00 +0000
+lang: en
+---
+
+Today, the canonical repository of the Ruby programming language was moved to Git from Subversion.
+
+The web interface for the new repository is [https://git.ruby-lang.org](https://git.ruby-lang.org), and is provided by cgit. We can keep the commit hash from the contributor on the Ruby repository directly.
```

Comment:
@hsbt 2nd paragraph is still very cryptic. Do you mean something like „Contributers to the SVN repository will also get commit right for the new repository.“ ???

Reply:
I don't get this, too. "Keep something from somebody" is to avoid telling somebody something. But as I read, the meaning of the sentence is more like "We can show the commit hash made by the contributor on the Ruby repository directly."

Reply:
I doubt the Git commit hash is what is meant. I would rather think „commit bit“ or „commit access“.